loss-detection-app/pipelines/object_detection/4/pipeline.json

```diff
@@ -1,14 +1,12 @@
 {
-    "name": "object_detection",
-    "version": 1,
     "type": "GStreamer",
-    "template":"v4l2src device=\"{source[path]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw !  queue leaky=upstream ! gvadetect inference-id=inf0 model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvaidentify tracker=true ! gvametaconvert converter=json method=detection source=\"{source[path]}\" name=\"jsonmetaconvert\" include-no-detections=true ! gvametapublish method=\"{destination[type]}\" address=\"{destination[host]}\" topic=\"{destination[topic]}\" timeout=1000 ! queue ! appsink name=appsink",
+    "template":"v4l2src device=\"{source[path]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw !  queue leaky=upstream ! gvadetect model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvaidentify ! gvametaconvert name=\"metaconvert\" source=\"{source[path]}\" add-empty-results=true ! gvametapublish name=\"destination\" ! queue ! appsink name=appsink",
```

Comment:
same as above, but change the request type to "device" and name the source element "source"

Reply:
Done